token/program-2022/src/state.rs

```diff
@@ -80,6 +118,52 @@ impl Pack for Mint {
     }
 }
 
+/// Type-Length-Value Entry, used to encapsulate all extensions contained within an account
+#[repr(C)]
+#[derive(Clone, Copy, Debug, Default, PartialEq)]
+pub struct TlvEntry<V> {
+    pub extension: Extension,
+    pub length: u32,
+    pub value: V,
+}
+
+/// Close authority extension data for mints.
+#[repr(C)]
+#[derive(Clone, Copy, Debug, Default, PartialEq)]
+pub struct MintCloseAuthorityExtension {
+    /// Optional authority to close the mint
+    pub close_authority: COption<Pubkey>,
+}
+
+/// Transfer fee information
+#[repr(C)]
+#[derive(Clone, Copy, Debug, Default, PartialEq)]
+pub struct TransferFee {
+    /// First epoch where the transfer fee takes effect
+    pub epoch: Epoch,
+    /// Amount of transfer collected as fees, expressed as basis points of the
+    /// transfer amount, ie.  increments of 0.01%
+    pub transfer_fee_basis_points: u16,
+    /// Maximum fee assessed on transfers, expressed as an amount of tokens
+    pub maximum_fee: u64,
+}
+
+/// Transfer fee extension data for mints.
+#[repr(C)]
+#[derive(Clone, Copy, Debug, Default, PartialEq)]
+pub struct MintTransferFeeExtension {
+    /// Optional authority to set the fee
+    pub transfer_fee_config_authority: COption<Pubkey>,
+    /// Harvest from mint instructions must be signed by this key
```

Comment:
```suggestion
    /// Withdrawal from mint instructions must be signed by this key
```
See my earlier comment. I think it's confusing to call this step harvesting sometimes.

Reply:
Definitely, the term `Harvest` is now only in the instruction to move withheld tokens from accounts to the mint